app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -79,6 +79,7 @@ import org.oppia.android.domain.onboarding.ExpirationMetaDataRetrieverModule
 import org.oppia.android.domain.oppialogger.LogStorageModule
 import org.oppia.android.domain.oppialogger.loguploader.LogUploadWorkerModule
 import org.oppia.android.domain.oppialogger.loguploader.WorkManagerConfigurationModule
+import org.oppia.android.domain.platformparameter.PlatformParameterModule
```

Comment:
Just want to confirm that you checked whether we actually need these changes? I have a strong suspicion that we do, but want to know if you tried running tests without the change. One way to quickly check is to remove it for one of the tests and see if it still passes.

Reply:
Yeah I have tried the way you told me to do. I ran the SplashTestActivityTest and then compilation for failed for different tests, so one by one I added this PlatformParameterModule in all the tests. There are no unnecessary changes .